tcpip/transport/tcp/connect.go

```diff
@@ -1086,6 +1093,16 @@ func (e *endpoint) disableKeepaliveTimer() {
 // goroutine and is responsible for sending segments and handling received
 // segments.
 func (e *endpoint) protocolMainLoop(handshake bool) *tcpip.Error {
+	op := ops.Begin("netstack_tcp_loop")
+	defer op.End()
```

Comment:
Just for my own personal edification: this won't actually do anything unless some reporter is hooked up to the `ops` right? Or is there some other thing that it does out of the box?

Reply:
Correct, it just for the reporter.

Reply:
thanks!